src/Process/FeatureProcessing.js

```diff
@@ -34,6 +34,21 @@ export default {
         }
 
         const features = node.children.filter(n => n.layer == layer.id);
+        for (const feat of features) {
```

Comment:
I don't agree too much, because we traverse even if no property has changed

Reply:
may be add a `layer._propertyChange` to notify change

Reply:
I believe it's good enough for now: it does the job and the perf hit from traversing a few nodes each frame will be unnoticable.

Reply:
I tend to agree with @peppsac for that: traversal cost for such simple operations are very cheap. Of course, profiling can prove us wrong.